ui/components/component-library/tag-url/tag-url.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from '../../ui/box/box';
+import { Text } from '../text';
+import {
+  ALIGN_ITEMS,
+  BORDER_RADIUS,
+  COLORS,
+  DISPLAY,
+  SIZES,
+  TEXT,
+} from '../../../helpers/constants/design-system';
+import { AvatarFavicon } from '../avatar-favicon';
+import { ButtonLink } from '../button-link';
+
+export const TagUrl = ({
+  label,
```

Comment:
Should `label` be required?

Reply:
we are using the `Text ` component here and the `label` is the prop we pass to show the text inside Tag Url.

For reference, `label` in this screenshot is set to `https://app.uniswap.org`
<img width="264" alt="Screenshot 2022-10-25 at 11 36 20 PM" src="https://user-images.githubusercontent.com/39872794/197848883-cc07eda6-6af3-45fb-b7f7-69b3489202cc.png">